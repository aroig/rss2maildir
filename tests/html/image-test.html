<h1>This just tests a simple image</h1>

<p>Images are a fundamental item, innit. <img src="http://www.example.com/smilie.jpg" alt=":)" /></p>
