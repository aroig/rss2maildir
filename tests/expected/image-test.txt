This just tests a simple image
==============================

Images are a fundamental item, innit. |:)|

.. |:)| image:: http://www.example.com/smilie.jpg
